installer/tue-install-impl.bash

```diff
@@ -1229,6 +1232,14 @@ stamp=$(date_stamp)
 INSTALL_DETAILS_FILE=/tmp/tue-get-details-$stamp
 touch "$INSTALL_DETAILS_FILE"
 
+for branch in $BRANCH
+do
+    tue-install-debug "Parsed branch $branch"
+    # Switch tue-env immediately to this branch:
+    _try_branch "$TUE_DIR" "$branch"
+    [[ -n $_try_branch_res ]] && tue-install-info "[tue-env] $_try_branch_res"
+done
```

Comment:
Everything regarding `tue-env` should be handled in `tue-install.bash`

Right now I also get:
```
/home/amigo/.tue/installer/tue-install-impl.bash: line 1239: _try_branch: command not found
```